src/plugins/intel_gpu/src/plugin/infer_request.cpp

```diff
@@ -554,7 +549,11 @@ void InferRequest::wait() {
             };
             auto layout = layout_by_rank(out_rank);
             auto tensorDesc = InferenceEngine::TensorDesc(precision, dims, layout);
-            _outputs[no.first] = create_host_blob(tensorDesc);
+            if (_outputs.find(no.first) == _outputs.end()) {
+                _outputs[no.first] = create_host_blob(tensorDesc);
+            } else {
+                _outputs[no.first]->setShape(dims);
```

Comment:
(out of curiosity) How is this enough? if the new dim is bigger than the original blob, will it reallocate?

Reply:
Yes, it reallocates internally

Reply:
Thanks!